src/components/app/app.tsx

```diff
@@ -13,12 +13,14 @@ import {
   Routes,
   Route
 } from 'react-router-dom';
+import { ReviewType } from '../../mock/reviews-mock';
```

Comment:
советую импортировать типы вот таким образом:
```suggestion
import type { ReviewType } from '../../mock/reviews-mock';
```

Таким образом, мы говорим сборщику, что не нужно оставлять эти файлы в продакшен билде. В результате, мы уменьшим размер продакшен билда